crates/environ/src/module_environ.rs

```diff
@@ -124,6 +125,7 @@ pub struct DebugInfoData<'a> {
     debug_loclists: gimli::DebugLocLists<Reader<'a>>,
     pub debug_ranges: gimli::DebugRanges<Reader<'a>>,
     pub debug_rnglists: gimli::DebugRngLists<Reader<'a>>,
+    pub dwarf_package: Option<DwarfPackage<Reader<'a>>>,
```

Comment:
Mind leaving a comment here indicating that this is not parsed from the wasm module and must be configured externally?

Either that or, alternatively, if it makes sense it might be best to thread the dwarf package as an auxiliary parameter through to the debuginfo processing. If there's too many layers to thread through though I think it's reasonable to leave it here.